src/atmega328p/com/usart.rs

```diff
@@ -0,0 +1,166 @@
+//     RustDuino : A generic HAL implementation for Arduino Boards in Rust
+//     Copyright (C) 2021   Kshitij Kaithal and Richa Prakash Sachan, Indian Institute of Technology Kanpur
```

Comment:
I need single names, they will be verified using the percentage of code written.

Reply:
fixed everywhere